<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fct_fournie_add</class>
 <widget class="QDialog" name="fct_fournie_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout BD V1</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_values">
    <item>
     <widget class="QLabel" name="label_values">
      <property name="text">
       <string>VALUES</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_values"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Add">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_indication">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>170</y>
     <width>621</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Indication : Completer les lignes correspondante pour former un INSERT valide </string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>821</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_insert">
    <item>
     <widget class="QLabel" name="label_insert">
      <property name="text">
       <string>INSERT INTO</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_insert"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_exemple">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Exemple : INSERT INTO LesSportifs_base VALUES ('1290','BERARD','Francois','USA','masculin','1996-01-02 00:00:00')</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_add_info">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>782</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_fct_add_query">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>782</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Add</sender>
   <signal>clicked()</signal>
   <receiver>fct_fournie_add</receiver>
   <slot>refreshResult()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refreshResult()</slot>
 </slots>
</ui>
